indico/modules/events/sessions/templates/management/session_person_list.html

```diff
@@ -0,0 +1,75 @@
+<div id="persons-list" class="i-box titled">
+    <div class"i-box-title">
+        <div class="toolbar">
+            <div class="group left">
+                <a class="icon-checkbox-checked i-button arrow js-dropdown"
+                   href="#" title="{% trans %}Select{% endtrans %}" data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="#" data-select-all="#persons-list input.select-row:visible">
+                            {% trans %}All{% endtrans %}
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" data-select-none="#persons-list input.select-row:visible">
+                            {% trans %}None{% endtrans %}
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="group i-selection">
+                <span class="i-button label">{% trans %}Show{% endtrans %}</span>
+                <input type="checkbox" id="filter-speakers" data-filter="speaker" checked>
+                <label for="filter-speakers" class="i-button">{% trans %}Speakers{% endtrans %}</label>
+                <input type="checkbox" name="authors" id="filter-primary" data-filter="primary" checked>
+                <label for="filter-primary" class="i-button">{% trans %}Primary authors{% endtrans %}</label>
+                <input type="checkbox" name="authors" id="filter-secondary" data-filter="secondary" checked>
+                <label for="filter-secondary" class="i-button">{% trans %}Co-authors{% endtrans %}</label>
+            </div>
+        </div>
+    </div>
+    <div>
+        <table class="i-table-widget">
+            <thead>
+                <tr>
+                    <th class="checkbox-column"></th>
+                    <th>{% trans -%}Name{%- endtrans %}</th>
+                    <th>{% trans -%}Email{%- endtrans %}</th>
+                    <th>{% trans -%}Affiliation{%- endtrans %}</th>
+                    <th>{% trans -%}Speaker{%- endtrans %}</th>
+                    <th>{% trans -%}Author type{% endtrans %}</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for session_person in session_persons -%}
+                    {% set event_person = session_person.person %}
+                    {% set is_primary_author = session_person.author_type.name == 'primary' %}
+                    {% set is_speaker = session_person.is_speaker %}
+                    <tr data-person-data="{{ 'speaker,' if is_speaker }}{{ 'primary' if is_primary_author else 'secondary' }}">
```

Comment:
and instead of `person_data` I'd call it `person_roles` ("person data" sounds more like name etc)